test-cases/longevity/longevity-1TB-3days-authorization-and-tls-ssl.yaml

```diff
@@ -0,0 +1,35 @@
+test_duration: 5020
+prepare_write_cmd: ["cassandra-stress write cl=ALL n=1100200300 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=150 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..1100200300"]
+
+stress_cmd: ["cassandra-stress mixed cl=QUORUM duration=4140m -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..550100150  -log interval=5 -col 'size=FIXED(200) n=FIXED(5)'",
+             "cassandra-stress read cl=QUORUM duration=4140m -mode cql3 native  -rate threads=10 -pop seq=550100151..1100200300  -log interval=5 -col 'size=FIXED(200) n=FIXED(5)'"]
```

Comment:
when returning this job to have 4 loaders, we need to split each (mixed and read) by 2, so we have 4 commands in total, so when we use round robin, each loader will run 1 command

Reply:
done